python/pdstools/utils/cdh_utils.py

```diff
@@ -137,6 +134,30 @@ def readDSExport(
     return import_file(file, extension, **reading_opts)
 
 
+def _is_url(path: str, filename: str):
```

Comment:
Please check for response.status_code == 200. That makes sure it's a _valid and working_ URL